sdks/universal-router-sdk/src/utils/routerCommands.ts

```diff
@@ -25,7 +25,8 @@ export enum CommandType {
   V4_SWAP = 0x10,
   V3_POSITION_MANAGER_PERMIT = 0x11,
   V3_POSITION_MANAGER_CALL = 0x12,
-  V4_POSITION_CALL = 0x13,
+  V4_INITIALIZE_POOL = 0x13,
+  V4_POSITION_MANAGER_CALL = 0x14,
```

Comment:
reason for changing command index for posm call? breaking change on the previous version

Reply:
oh damn didnt see this 🤦‍♀️ mm i dont think there was a a good reason... so probably shouldnt have happened. Thats on me